src/zenml/new/pipelines/pipeline.py

```diff
@@ -691,6 +693,33 @@ def _run(
 
             self.log_pipeline_deployment_metadata(deployment_model)
 
+            # TODO: We should probably only do this if there is no schedule?
+            # It will still work, but it might be hours/days until the run gets
+            # populated
+            pipeline_run_request = PipelineRunRequestModel(
+                name=get_run_name(
+                    run_name_template=deployment_model.run_name_template
+                ),
+                orchestrator_run_id=None,
+                user=Client().active_user.id,
+                workspace=Client().active_workspace.id,
+                deployment=deployment_model.id,
+                pipeline=deployment_model.pipeline.id
+                if deployment_model.pipeline
+                else None,
+                # TODO: Maybe we want a state called `preparing` or `launching`
+                # to differentiate between waiting for the first step to get
+                # started
+                status=ExecutionStatus.RUNNING,
+                # Is this the actual start time now, or do we want to set this
+                # once the first step started? Especially for scheduled runs
+                # this would be strange
+                start_time=datetime.utcnow(),
```

Comment:
Depends how we want to define the execution time of a run - does it make sense for this to include the image building / pushing times or should it only be the cummulative execution times of all steps? Maybe @htahir1 has an opinion here?

In any case, we already store `created` as a separate field in the DB so we will also always know when this object was created even if we set the `start_time` later.

Also, scheduling can be ignored IMO since we shouldn't write such placeholder runs for scheduled runs anyway (see comment above).

Reply:
I would argue that we should not use the time here as the start time to keep it consistent with our other runs. Scheduled runs (and reruns) will have their start time set to when the first step starts executing, so I think that should also be the case for placeholder runs.

Reply:
While I would still prefer to set the start date only once the first step starts, I think it's nicer to have this timestamp match the one which is used in the run name.

For this reason we decided to go with the simplest approach, which is to simply set both `run_name` and `start_date` already when creating the placeholder run, and not updating it down the line.